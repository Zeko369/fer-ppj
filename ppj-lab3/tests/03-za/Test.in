<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 2
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <za_petlja>
     KR_ZA 2 za
     IDN 2 i
     KR_OD 2 od
     <E>
      <T>
       <P>
        BROJ 2 1
       <T_lista>
        $
      <E_lista>
       $
     KR_DO 2 do
     <E>
      <T>
       <P>
        BROJ 2 5
       <T_lista>
        $
      <E_lista>
       $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 3 x
        OP_PRIDRUZI 3 =
        <E>
         <T>
          <P>
           IDN 3 x
          <T_lista>
           OP_PUTA 3 *
           <T>
            <P>
             IDN 3 i
            <T_lista>
             $
         <E_lista>
          $
      <lista_naredbi>
       $
     KR_AZ 4 az
   <lista_naredbi>
    $
