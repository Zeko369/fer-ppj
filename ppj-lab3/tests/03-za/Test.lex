IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 2
KR_ZA 2 za
IDN 2 i
KR_OD 2 od
BROJ 2 1
KR_DO 2 do
BROJ 2 5
IDN 3 x
OP_PRIDRUZI 3 =
IDN 3 x
OP_PUTA 3 *
IDN 3 i
KR_AZ 4 az
