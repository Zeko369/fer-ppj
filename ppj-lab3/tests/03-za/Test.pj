x = 2 
za i od 1 do 5
 x = x * i
az
