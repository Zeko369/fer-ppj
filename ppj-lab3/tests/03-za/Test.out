3 1 x
3 2 i
